src/main/java/com/todolist/enterprise/TaskController.java

```diff
@@ -7,14 +7,17 @@
 
 @Controller
 public class TaskController {
-
+    /**
+     * Manages root endpoint and return to home page
+     * @return Home page
+     */
```

Comment:
Added Javadoc
[Short summary of Java Coding Pactices](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)